packages/web-react/src/types/shared/style.ts

```diff
@@ -12,6 +12,10 @@ export type SpacingProps = {
     | Partial<Record<BreakpointToken, SpaceToken | StyleSpacingAuto>>;
 };
 
+export interface ComponentSpacingProps {
```

Comment:
What about just `SpacingProp` since it has only one attribute?

Reply:
As I wrote, SpacingProps already exists and I can't use it

Reply:
`SpacingProp` vs `SpacingProps` note the `s` on the end. The first one does not exist in our project.

Reply:
> `SpacingProp` vs `SpacingProps` note the `s` on the end

Did anyone say "mind the good naming habbits"? 🙂